src/menu.tsx

```diff
@@ -154,6 +154,16 @@ export function get_menu_items(
                     onClick: () => downloadFile(path, item)
                 }
             );
+        } else if (item.type === "dir" && cockpit.manifests.system?.tools?.terminal?.capabilities?.includes("path")) {
+            menuItems.push(
+                { type: "divider" },
+                {
+                    id: "terminal",
+                    title: _("Open in terminal"),
```

Comment:
Does this only work for the context menu of a selected subdirectory, or also in the context menu of the currently shown directory? I. e. if you click in an empty area. That almost feels like the more important use case. Either way both of these should be tested.

Reply:
Added!